cloud/vmware/vsphere_guest.py

```diff
@@ -79,6 +79,12 @@
     description:
       - Name of the source template to deploy from
     default: None
+  snapshot_to_clone:
+    version_added 2.0
```

Comment:
Need to put the 2.0 in quotes otherwise the doc formatter thinks that 2.0 is a float instead of a string.